src/appl/rcsfinteract90/set_CSF_list.f90

```diff
@@ -86,10 +87,13 @@ SUBROUTINE SET_CSF_list(NCORE,NPEEL)
 !
       READ (21, '(A)') S_orbitals_1
       READ (20, '(A)') S_orbitals_2
-!      I = LEN_TRIM(S_orbitals_1)
-!      IF(S_orbitals_1(1:I) /= S_orbitals_2(1:I)) then
-!         STOP "Different order of Peel orbitals"
-!      end if
+      I = LEN_TRIM(S_orbitals_1)
+      IF(S_orbitals_1(1:I) /= S_orbitals_2(1:I)) then
+         WRITE (6, *) ''
+         WRITE (6, *) 'Error in input !!!'
+         WRITE (6, *) 'Different order of Peel orbitals'
+         STOP ' in files rcsfmr.inp and rcsf.inp'
```

Comment:
I don't quite remember Fortran's behaviour here, but maybe have this be `ERROR STOP` to make sure that the program returns a non-zero exit code?

```suggestion
         ERROR STOP ' in files rcsfmr.inp and rcsf.inp'
```

Reply:
I googled it and Morten seems to be right: the "ERROR STOP" ( tested with gfortran and ifort ) is the thing to do, i.e. an error termination instead of a normal termination. Moreover, I read that "ERROR STOP" has more severe consequences in parallel programs than a simple stop. When executed on one coarray image (processor, thread,...), it initiates error termination there and hence causes all other images that have not already initiated error termination to initiate error termination. It causes the whole calculation to stop as soon as is possible. Therefore, error stop should be used in parallel programming when a severe error in one of the images makes the entire simulation useless for the rest of images.

Reply:
I'm fine with either - we don't use coarrays though, and I'm not sure how MPI deals with error stop vs stop - in a way it's not really a code error here, it's more a question of an error at the user side of things, so for me, a normal termination with information is just a fine and give a less cluttered response to the user at runtime. But to be safe, let's just use error stop in general. Great!